### ISSUE TITLE
Error when initializing LIST DoctorsInSurgery = (Adams, Bernard) ? Intended?

### ISSUE THREAD

**Issue by CarterG81** [inkle/ink#393]
The only way to initialize a list is by having parathesis ( ) around each INDIVIDUAL value.

LIST DoctorsInSurgery = (Adams), (Bernard) //Success

When modifying the list, this works:

~ DoctorsInSurgery = (Adams, Bernard) //Success

However when initializing, we can't do this:

LIST DoctorsInSurgery = (Adams, Bernard) //ERROR!

Is there a reason why the syntax this way? I think I am beginning to understand why (it's starting to make sense as I type this out) but just wanted to make sure this wasn't overlooked. It's easier to type just two ( ) around the entire LIST (ex. if you wanted to make ALL doctors in) rather than do each one individually (which is a lot of extra work if you have a huge LIST).

**Comment by joethephish** [comment]
Yeah, ideally that syntax would be possible, I just didn't get round to it, sorry!

**Comment by joningold** [comment]
One for the community to tackle maybe? It’d certainly be a useful syntax.
